plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -73,55 +97,56 @@ When a plugin is updated, its activation function registered with `register_acti
 
 The `plugins_loaded` action hook is a great choice for this because it is triggered right after all active plugins have been loaded. This ensures the database tables are updated before any other code tries to query against them.
 
-To track which database version is already installed, we'll store a simple database version number for the plugin in the options table.
+To track which database version is already installed, you can store a simple database version number for the plugin in the options table.
 
 ```php
-global $my_plugin_db_version;
-$my_plugin_db_version = '1.0';
-
-add_action( 'plugins_loaded', 'my_plugin_update_db_check' );
-function my_plugin_update_db_check() {
-	global $my_plugin_db_version;
-	if ( get_option( 'my_plugin_db_version' ) != $my_plugin_db_version ) {
-		my_plugin_create_table();
+$wp_learn_custom_table_db_version = '1.0';
+```
+
+```
```

Comment:
Should these be combined? If not, the second code block is missing the `php` type. However, I do think the first code snippet feels kind of lost and floating if it isn't in a combined code block with the rest of the code. That code line could be anywhere, where if combined with the next code block, it seems safe to assume it's being initialized in the global namespace, outside of any function.

Reply:
You're not wrong. The main reason I separated them is that I'm also thinking of how this will present in the video lesson. I'm going to leave this comment open for now, and review it later today.

Reply:
Adding language in between makes much more sense now. This is resolved. Thank you!